Makefile

```diff
@@ -1,32 +1,13 @@
 ROOT:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-
 IMAGE_NAME:=jenkins4eval/ssh-slave
-IMAGE_ALPINE:=${IMAGE_NAME}:alpine
-IMAGE_DEBIAN:=${IMAGE_NAME}:test
-IMAGE_JDK11:=${IMAGE_NAME}:jdk11
-
-build: build-alpine build-debian build-jdk11
-
-build-alpine:
-	docker build -t ${IMAGE_ALPINE} --file Dockerfile-alpine .
-
-build-debian:
-	docker build -t ${IMAGE_DEBIAN} --file Dockerfile .
-	
-build-jdk11:
-	docker build -t ${IMAGE_JDK11} --file Dockerfile-jdk11 .
 
-.PHONY: test
-test: test-alpine test-jdk11 test-debian
+build:
```

Comment:
I don't agree with this change. Sometimes it is useful to be able to build just one platform for testing.